kuma/demos/tests/test_helpers.py

```diff
@@ -14,9 +12,9 @@ def test_tag_description_no_description(self):
             "title": "Testing tag without description",
         }
         description = tag_description("test_tag")
-        eq_("test_tag", description)
+        self.assertEqual("test_tag", description)
```

Comment:
I like this here, but are we setting a precedent to remove nose `eq_` everywhere?